### PR TITLE
volcano metrics source support elasticsearch

### PR DESCRIPTION
**Pull request by Shoothzj**
See #2679 for details.
This PR mainly adds the capability to fetch Elasticsearch metrics to Volcano, enabling it to integrate with a wider range of monitoring systems.

### CODE REVIEW

**Comment by gj199575** [comment]
Reviewed

File: pkg/scheduler/metrics/source/metrics_client_elasticsearch.go [hunk 1]
```diff
@@ -0,0 +1,93 @@
+package source
```
Comment (william-wang) [inline pkg/scheduler/metrics/source/metrics_client_elasticsearch.go:1]: please add the copyright.

Comment (Shoothzj) [reply pkg/scheduler/metrics/source/metrics_client_elasticsearch.go:17]: fixed

**Review by william-wang — APPROVED** [review]
/approve

**Pull request merged**